docs/elements.md

```diff
@@ -899,7 +899,7 @@ Provides an in-browser rich text editor, aimed mostly at manual grading essay-ty
 | `source-file-name`   | string                              | —                    | Name of the source file with existing content to be displayed in the editor. The format of this file must match the format specified in the `format` attribute.                                                                                                                                                                                                                                                                                           |
 | `directory`          | string                              | See description      | Directory where the source file with existing code is to be found. Only useful if `source-file-name` is used. If it contains one of the special names `"clientFilesCourse"` or `"serverFilesCourse"`, then the source file name is read from the course's special directories, otherwise the directory is expected to be in the question's own directory. If not provided, the source file name is expected to be found in the question's main directory. |
 | `placeholder`        | string                              | `"Your answer here"` | Text to be shown in the editor as a placeholder when there is no student input.                                                                                                                                                                                                                                                                                                                                                                           |
-| `format`             | `"html"` or `"markdown"`            | `"html"`             | Format used to save the student's response. This format also affects how the source file name or inner HTML is interpreted.                                                                                                                                                                                                                                                                                                                               |
+| `format`             | `"html"` or `"markdown"`            | `"html"`             | Format used to interpret the source file name or starting content. This option does not affect the output format.                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Nit, it's about the file contents, not the file's name. I'd also be happy dropping the word "specified" from here.

```suggestion
| `format`             | `"html"` or `"markdown"`            | `"html"`             | Format used to interpret the specified source file or starting content. This option does not affect the output format.                                                                                                                                                                                                                                                                                                                                         |
```